script/scan-build

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+. script/lib.sh
+
+scan_build make "$@"
```

Comment:
🤔 This was useful for quick testing but you are right, it isn't particularly useful now.